client/src/main/test/com/battleships/gamewindow/services/BufforCalculatorTest.java

```diff
@@ -0,0 +1,75 @@
+package com.battleships.gamewindow.services;
+
+import com.battleships.Coordinate;
+import com.battleships.gamewindow.board.PlayerBoard;
+import com.battleships.gamewindow.board.Ship;
+import com.battleships.gamewindow.board.fieldStates.BoardField;
+import com.battleships.gamewindow.board.fieldStates.FieldState;
+import javafx.embed.swing.JFXPanel;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.testng.Assert.assertEquals;
+
+
+@Test
+public class BufforCalculatorTest {
+    private JFXPanel jfxPanel = new JFXPanel();
+    private PlayerBoard playerBoard;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        playerBoard = new PlayerBoard();
+        initBoardWithSea();
+    }
+
+    private void initBoardWithSea() {
+        for (int row = 1; row <= 10; row++) {
+            for (int col = 1; col <= 10; col++) {
+                Coordinate coordinate = Coordinate.fromIntCoords(row, col);
```

Comment:
All these three statements could be in one method call.

Reply:
You could chainge it to IntStream

Reply:
voilà